test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -87,6 +91,31 @@ namespace iroha {
       std::string block_store_path = "/tmp/block_store";
     };
 
+    TEST_F(AmetsuchiTest, GetBlocksCompletedWhenCalled) {
+      // Commit block => get block => observable completed
+      auto storage =
+          StorageImpl::create(block_store_path, redishost_, redisport_, pgopt_);
+      ASSERT_TRUE(storage);
+
+      model::Block block;
+      block.height = 1;
+
+      auto ms = storage->createMutableStorage();
+      ms->apply(block, [](const auto &blk, auto &executor, auto &query,
+                          const auto &top_hash) { return true; });
+      storage->commit(std::move(ms));
+
+      auto times_wrapper =
+          make_test_subscriber<CallExact>(storage->getBlocks(1, 1), 1);
+      times_wrapper.subscribe();
+      ASSERT_TRUE(times_wrapper.validate());
+
+      auto completed_wrapper =
+          make_test_subscriber<IsCompleted>(storage->getBlocks(1, 1));
```

Comment:
This functionality looks like another test. Pls, add description for  expectations about `CallExact` or separate on two tests.